radish/documentors/__main__.py

```diff
@@ -0,0 +1,211 @@
+# -*- coding: utf-8 -*-
+
+"""
+This module provides a script to document Radish step implementations
+"""
+
+from sys import exit
+
+from os.path import split
+
+from docopt import docopt
+
+from radish import __VERSION__
+
+from radish.documentors.sphinx.multipage import MultiPageSphinxDocumentSet
+from radish.documentors.sphinx.singlepage import SinglePageSphinxDocument
+from radish.documentors.steps_writer import StepsDocumentWriter
+
+from radish.errororacle import error_oracle, enable_excepthook
+
+from radish.exceptions import RadishError
+
+from radish.loader import load_module_assets
+
+from radish.main import setup_config
+
+from radish.stepregistry import StepRegistry
+
+
+# ........................................................................... #
+@error_oracle
+def main():
+    usage = """
+Usage:
+    radish-docs sphinx <module-path>
+        [-o=<folder-path> | --output-folder=<folder-path>]
+        [--index-file-name=<file-path> ]
+        [-s=<file-path> | --single-file=<file-path>]
+        [-c | --print-to-console]
+        [--label-prefix=<label-prefix>]
+        [-r | --recreate]
```

Comment:
> To me update implies modification of existing state (file or folder)

Agreed.

> Another possible name is `--overwrite`

I'd prefer `--overwrite` over `--update`. I suggest to go with that one for now.